packages/react-ui/components/Tabs/__stories__/Tabs.stories.tsx

```diff
@@ -480,3 +489,48 @@ TabsInModalStory.story = { name: 'tabs in modal', parameters: { creevey: { skip:
 
 export const HoverTable = () => <TabsTable />;
 HoverTable.story = { name: 'hover table', parameters: { creevey: { skip: [true] } } };
+
+export const TabsWithImage: CSFStory<JSX.Element> = () => {
+  const [activeTab, setActiveTab] = React.useState('search4');
+
+  return (
+    <Tabs value={activeTab} vertical onValueChange={tab => setActiveTab(tab)}>
+      <Tabs.Tab id="search1">
+        <Img size={'75px'} />
+      </Tabs.Tab>
+      <Tabs.Tab id="search2">
+        <Img size={'105px'} />
+      </Tabs.Tab>
+      <Tabs.Tab id="search3">
+        <Img size={'25px'} />
+      </Tabs.Tab>
+      <Tabs.Tab id="search4">
+        <Img size={'150px'} />
+      </Tabs.Tab>
+      <Tabs.Tab id="search5">
+        <Img size={'30px'} />
+      </Tabs.Tab>
+      <Tabs.Tab id="search6">
+        <Img size={'100px'} />
+      </Tabs.Tab>
+    </Tabs>
+  );
+};
+TabsWithImage.story = {
+  name: 'Tabs with images',
+  parameters: {
+    creevey: {
+      tests: {
+        async ['After render with images']() {
+          await this.browser
+            .actions({
+              bridge: true,
+            })
+            .pause(500)
+            .perform();
+          await this.expect(await this.takeScreenshot()).to.matchImage('after render with images');
+        },
+      },
```

Comment:
В creevey есть специальный параметр - задержка перед выполнением всего теста. Примерно для таких ситуаций.

```suggestion
      delay: 500
```